icons/fidoalliance.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 24 24"><title>Fido Alliance icon</title><path d="M8.027 7.411a1.11 1.11 0 1 0 1.11 1.111v-.001a1.11 1.11 0 0 0-1.11-1.11zM5.054 10.52v1.45h1.993v4.902h.72v-2.786h.521v2.786h.726V10.52h-3.96zM14.442 11.068h-.022c-.293-.425-.881-.718-1.676-.718-1.523 0-2.86 1.251-2.85 3.394 0 1.98 1.208 3.274 2.72 3.274.816 0 1.595-.369 1.979-1.063l.12.913h1.707V7.553h-1.978zm0 2.97c0 .162-.012.32-.035.475l.002-.018a1.18 1.18 0 0 1-1.156.946h-.014.001c-.816 0-1.348-.674-1.348-1.762 0-1.001.456-1.808 1.359-1.808.62 0 1.045.447 1.164.969.015.092.023.198.023.307l-.001.056V13.2zM4.911 8.819l.486-1.636a2.86 2.86 0 0 0-.393-.103l-.02-.003a3.861 3.861 0 0 0-1.956.067l.027-.007a2.05 2.05 0 0 0-.389.157l.012-.006c-.26.132-.484.294-.678.486-.511.501-.771 1.181-.771 2.02v.724h-1.23v1.45h1.226v4.902H3.19v-4.902h1.226V10.52H3.19v-.767a1.045 1.045 0 0 1 .235-.729l-.001.002a.77.77 0 0 1 .065-.063l.001-.001s.423-.419 1.421-.144zM20.767 10.351c-2.055 0-3.402 1.315-3.402 3.382s1.425 3.284 3.285 3.284c1.701 0 3.35-1.063 3.35-3.402-.002-1.916-1.296-3.264-3.233-3.264zm-.065 5.242c-.851 0-1.305-.816-1.305-1.914 0-.936.369-1.903 1.305-1.903.891 0 1.262.957 1.262 1.893 0 1.163-.493 1.925-1.262 1.925z"/></svg>
```

Comment:
Should probably include the "TM" that is found on their logo

Reply:
Fair enough. Added.